Sohail-sohailkamdar/Homework-3/json-homework/testscript.js

```diff
@@ -0,0 +1,46 @@
+
+
+
+let origins="24 Sussex Drive Ottawa ON"
+origins = origins.split(' ').join('+')
+let destination = "1461 Alice Street Oakland CA"
+destination = destination.split(' ').join('+')
+let mode = 'driving'
+
+let Url = makeURL(origins,destination)
+let json_obj = JSON.parse(Get(Url));
+console.log(json_obj)
+console.log(timeToDestination(json_obj))
+
+
+function makeURL(origins, destination){
+const key = "AIzaSyAoII95Lhck_6i3XJHBfQefjxqDqHCN358"
```

Comment:
you definitely want to make sure you dont commit this api key to a public repo - lets remove this and re-submit